src/main/java/mate/academy/internetshop/securiti/AuthenticationServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package mate.academy.internetshop.securiti;
+
+import mate.academy.internetshop.exception.AuthenticationException;
+import mate.academy.internetshop.lib.Inject;
+import mate.academy.internetshop.model.User;
+import mate.academy.internetshop.service.Service;
+import mate.academy.internetshop.service.UserService;
+
+@Service
+public class AuthenticationServiceImpl implements AuthenticationService {
+    @Inject
+    private UserService userService;
+
+    @Override
+    public User login(String login, String password) throws AuthenticationException {
+        User userFromDB = userService.findByLogin(login).orElseThrow(() ->
+                new AuthenticationException("Incorrect login"));
+        if (!userFromDB.getPassword().equals(password)) {
+            throw new AuthenticationException("Incorrect password");
```

Comment:
```suggestion
            throw new AuthenticationException("Incorrect login or password");
```
краще не вказувати, що саме неправильно адже так буде менш сек'юрно (легше підібрати пароль до акаунта, коли знаєш, що логін коректний)

Reply:
changed